.github/workflows/publish-debug-image.yaml

```diff
@@ -33,13 +34,21 @@ jobs:
           username: ${{ secrets.GH_DOCKER_ACCESS_USER }}
           password: ${{ secrets.GH_DOCKER_ACCESS_TOKEN }}
 
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_REGISTRY_USER }}
+          password: ${{ secrets.QUAY_REGISTRY_PASSWORD }}
+
       - name: Docker meta
         id: meta
         uses: docker/metadata-action@v4
         with:
           # list of Docker images to use as base name for tags
           images: |
             ${{ env.DOCKER_REPOSITORY }}
+            ${{ env.QUAY_IMAGE }}
```

Comment:
I'd make either the previous variable be called DOCKER_IMAGE or make this variable be called QUAY_REPOSITORY, whichever you think is more accurate. I realize that these are under the `images` spec so perhaps that would be more accurate?

Reply:
Yes, you're right. Changing it to `QUAY_REPOSITORY`.